src/Bonsai.ML.Lds.Torch/SmoothedState.cs

```diff
@@ -0,0 +1,29 @@
+using static TorchSharp.torch;
+
+namespace Bonsai.ML.Lds.Torch;
+
+/// <summary>
+/// Represents the state of a Kalman smoother.
+/// </summary>
+/// <param name="smoothedMean"></param>
+/// <param name="smoothedCovariance"></param>
+public struct SmoothedState(
```

Comment:
Is there a reason we need this specific type? Looks like we might be able to simply use the `LdsState` class to represent the smoothed state.